lib/Db/CollectedAddressMapper.php

```diff
@@ -119,9 +119,11 @@ public function deleteOrphans(): void {
 		$result->closeCursor();
 
 		$qb2 = $this->db->getQueryBuilder();
-		$query = $qb2
-			->delete($this->getTableName())
-			->where($qb2->expr()->in('id', $qb2->createNamedParameter($ids, IQueryBuilder::PARAM_INT_ARRAY), IQueryBuilder::PARAM_INT_ARRAY));
-		$query->execute();
+		foreach (array_chunk($ids, 1000) as $chunk) {
+			$query = $qb2
```

Comment:
Suggestion: Can we move the query builder outside the foreach and use createParameter and setParameter like in MessageMapper?